src/constants.ts

```diff
@@ -49,7 +49,39 @@ export const IOS = Object.freeze({
   STOREKIT: 'ios.permission.STOREKIT' as const,
 });
 
-export const PERMISSIONS = Object.freeze({ANDROID, IOS});
+export const WINDOWS = Object.freeze({
+  APPOINTMENTS: 'appointments' as const,
```

Comment:
Could you use a format identical to other platforms? (`windows.permission.APPOINTMENTS` etc.) It's easier to read if you use TS typing instead of importing the `PERMISSIONS` object.

`RNPermissions.check("appointments")` will be 100% valid, type-checked code, but id not clear in a codebase that might target 3 operating systems.